packages/kit-headless/src/components/select/select.tsx

```diff
@@ -128,7 +133,8 @@ export const SelectTrigger = component$(
         ref={ref}
         aria-expanded={contextService.isExpanded.value}
         disabled={disabled}
-        onClick$={() => {
+        onClick$={(e) => {
+          e.stopPropagation();
```

Comment:
Question: Why do we need to use `stopPropagation`?

I believe there is a good reason for this.
Maybe we can extract this inline-block to a function and add a comment providing a bit more context why this is needed.

Reply:
So when there is any content that's currently in the select and you click on that content the select component will not toggle isExpanded.

I believe this has something to do with a parent event "intercepting" our other event. Because when I added e.stopPropagation it would actually toggle isExpanded when clicking on the content.

The only other solution I could think of for this would be to add pointer-events: none in CSS, but I think that would cause a lot of confusion on the consumer side as well as potential issues.